### ISSUE TITLE
Turn off database refresh in the background

### ISSUE THREAD

**Issue by elidourado** [devonzuegel/ketchup-club#11]
Since we're now a background app, we'll use a lot of unnecessary data and cycles if we continue to sync with the database while the app is in the background. So we need to turn that off and then back on again once we come to the foreground. Relevant code is in Home.js but I couldn't grok it at a glance.

**Comment by devonzuegel** [comment]
Agreed. This part of the codebase is a total mess, just a brute force implementation for now really. Definitely open to improving it!
